### ISSUE TITLE
[BUG] Rancher container unable to resolve due to chart version for rancher-webhook

### ISSUE THREAD

**Issue by thaneunsoo** [rancher/rancher#39073]
**Rancher Server Setup**
- Rancher version: v2.7-head
- Installation option (Docker install/Helm Chart): Docker
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc):
- Proxy/Cert Details:

**Information about the Cluster**
- Kubernetes version:
- Cluster Type (Local/Downstream):
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
  - If custom, define the set of permissions:

**Describe the bug**

**To Reproduce**

1. On Ubuntu 22.04 instance install Rancher using docker

**Result**
Rancher container is stuck trying to find rancher-webhook chart and is stuck repeating the following error message in the logs
```
[ERROR] available chart version (1.0.5+up0.2.6) for rancher-webhook is less than the min version (1.0.6+up0.2.7-rc4) 
2022/09/21 21:33:09 [ERROR] Failed to find system chart rancher-webhook will try again in 5 seconds: no chart name found
```

**Expected Result**

Rancher should run successfully

**Comment by MbolotSuse** [comment]
## Validation Template

## Root Cause

During the split between 2.6 and 2.7, there was a version of the webhook merged which is only available for 2.6 at the moment. 2.7 did not have a record of this chart, and was unable to deploy the webhook from the `dev-v2.7` branch of the charts.

## What was fixed, or what change have occurred

A new 2.7-specific version of the webhook was created (`0.3.0-rc2`) and made available in the 2.7 branch of the charts. In addition, rancher was configured to use this new version in the 2.7 branch.

## Areas or cases that should be tested

Basic install of the rancher-webhook. You can validate this by making sure that there is a deployment in the local cluster for the webhook deployment (`Workload` -> `Deployments` -> Cattle-system namespace), and that the version is as expected `0.3.0-rc2`.

## What areas could experience regressions

There may be regressions in the webhook due to changes made in recent webhook RCs. However, this should not impact basic install functionality. 

## Are the repro steps accurate/minimal?

Yes. They are included here for convenience. (Note: these steps come from the issue - a quick test on my local did not produce the issue). 

1. Run rancher
2. Observe an error message in the logs.

## Notes

You will have to wait for a new 2.7-head to be built before you can test.

**Comment by dasarinaidu** [comment]
I have completed validations on this ticket and checked with Michel B : He is Ok with the test coverage and validation parts. 

**Covered below validations:**
1. Rancher up with Docker -- rancher version v2.7-head
    a. Rancher is Up
    b. NO boot strap errors in the Logs related to chatversion(  root@localhost:~# docker logs  98ed085719b6  2>&1 | grep "available chart version" ) -- NO output
2. Rancher UI : I can see latest version of Webhook version ( rancher-webhook:2.0.0+up0.3.0-rc3 )
    a. Observation : Ticket mentioned the version rancher-webhook:2.0.0+up0.3.0-rc2 , I checked with Mike and he mentioend there was another PR by kevin with rc3 and rancher-webhook:2.0.0+up0.3.0-rc3 is the latest version

**Note :** To making sure my tests are good --- I also reproduce this issue with v2.7-bb9012ee0e66196136403d026b9e38c757c06c8c-head
      a. I saw boot logs has errors with "2022/10/03 20:39:06 [ERROR] available chart version (1.0.5+up0.2.6) for rancher-webhook is less than the min version (1.0.6+up0.2.7-rc4)"
      b. Rancher UI (Webhook version) shows :   rancher-webhook:1.0.5+up0.2.6 (edited)
